lib/ansible/modules/cloud/atomic/atomic_image.py

```diff
@@ -87,9 +98,32 @@ def core(module):
     image = module.params['name']
     state = module.params['state']
     started = module.params['started']
+    storage = module.params['storage']
     is_upgraded = False
 
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
+    out = {}
+    err = {}
+    rc = 0
+
+    # if storage is set, only pull the image
+    if storage:
+        if state == 'present' or state == 'latest':
+            args = ['atomic', 'pull', "--storage=%s" % storage, image]
+            rc, out, err = module.run_command(args, check_rc=False)
+            if rc < 0:
+                module.fail_json(rc=rc, msg=err)
+            else:
+                changed = "Writing manifest" in out or "Pulling layer" in out
+                module.exit_json(msg=out, changed=changed)
+        else:
+            args = ['atomic', 'images', 'delete',  "--storage=%s" % storage, image]
+            if rc < 0:
+                module.fail_json(rc=rc, msg=err)
+            else:
+                changed = "Unable to find" not in out
+                module.exit_json(msg=out, changed=changed)
+        return
```

Comment:
thinking more of it, there is no reason why they are mutually exclusive, I've changed it to handle 'started' when state is 'present' or 'latest'.